README.md

```diff
@@ -30,6 +30,13 @@ find . -name "*_shpec.sh" | entr shpec
 [shpec's own tests](https://github.com/rylnd/shpec/tree/master/shpec/shpec_shpec.sh)
 are a great place to start. More detailed examples to come.
 
+### Mocking
```

Comment:
We could Try waffle.io then. 
It was to discuss the feature I start to add, and keep track of the ideas, and one we should add now or then. (this is extra stuff, no issue ^^)